### DISCUSSION TITLE
an issue with SMTP on WIKIjs

### DISCUSSION THREAD

**Discussion by SATadsl** [requarks/wiki#5524] [Error / Bug Report]
Hello , 

I really have an issue using SMTP on WIKIJS while I'm trying sending an email I receive this error 

140452020725696:error:141A318A:SSL routines:tls_process_ske_dhe:dh key too small:../deps/openssl/openssl/ssl/statem/statem_clnt.c:2166:

I really need your help fixing this issue.

Best Regards

**Comment by DarkArc** [comment]
Duplicate of https://github.com/requarks/wiki/discussions/3574

Edit: Maybe not this one, oops.
